osbs/api.py

```diff
@@ -475,7 +475,6 @@ def _do_create_prod_build(self, git_uri, git_ref,
             koji_kerberos_keytab=self.build_conf.get_koji_kerberos_keytab(),
             koji_kerberos_principal=self.build_conf.get_koji_kerberos_principal(),
             architecture=architecture,
-            platform=platform,
```

Comment:
No, this is in the set_params() call where we already set the platform node selector. We don't need the platform for anything else.